crates/model/src/solver_competition.rs

```diff
@@ -23,6 +23,17 @@ pub struct Request {
     pub transaction: Transaction,
     pub competition: SolverCompetitionDB,
     pub executions: Vec<(OrderUid, Execution)>,
+    pub scores: Scores,
+    pub participants: Vec<H160>, // solver addresses
+}
+
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
+pub struct Scores {
+    #[serde(with = "u256_decimal")]
+    pub winning_score: U256,
+    #[serde(with = "u256_decimal")]
+    pub reference_score: U256,
+    pub block_deadline: u64,
 }
```

Comment:
If the auction participants are no longer an array, we may have to record the winning solver here.

Reply:
Putting the winner here is cleaner and more explicit.

Reply:
Added the winner in `scores` table.